DuckDuckGo/ContentBlocker/ContentBlocking.swift

```diff
@@ -60,16 +60,16 @@ final class AppContentBlocking {
     // keeping whole ContentBlocking state initialization in one place to avoid races between updates publishing and rules storing
     @MainActor
     init(internalUserDecider: InternalUserDecider) {
-        let configStorage = ConfigurationStore.shared
+        let configStorage = ConfigurationStore()
```

Comment:
Should we inject the `ConfigurationStore` instance in the initializer, from the one that gets created in the app delegate?